.github/olm-publish/action.yaml

```diff
@@ -52,7 +50,7 @@ runs:
       # make initiate-release conflicts with the previous bundle created for
       # development branch, so the version in main takes precedence over the
       # one in olm-catalog branch
-      git merge -Xours origin/olm-catalog  -m "ci: merge olm-catalog branch"
+      git merge --allow-unrelated-histories -Xours origin/olm-catalog  -m "ci: merge olm-catalog branch"
```

Comment:
note: this was required to get my CI passing - 
See failure - https://github.com/sthaha/observability-operator/actions/runs/5408840544/jobs/9828324412